docs/core-services/container-orchestration-provider-usage.md

```diff
@@ -137,3 +139,23 @@ After the starting phase just described, Kura will continuously monitor the acti
 
 ![Container Orchestration Provider Enfrocement Multiple Digests](./images/container-orchestration-provider-enforcement-multiple-digests.png)
 
+
+## Container Instance Enforcement
+
+In the [Container Instance configuration](#configuring-the-container) is included an option that allows to add a specific digest to the enforcement allowlist: this option was designed to allow the authorization of a specific image, but only when a Container Instance run by Kura is using it.
+
+Everytime the enforcement feature performs a check on its startup or when a new container is started, it will consider as authorized all the digests included in the Container Orchestration Allowlist and those provided by the enabled Container Instances in Kura through the *Container Image Enforcement Digest* option.
+
+
+So the authorization schemas presented in the [previous section](#container-enforcement) are updated like:
+![Enforcement Flow With Instances](./images/diagramAllowlistWithInstances.png)
+
+As you can see, the *ContainerInstance2* digest is not included in the final allowlist, because it is disabled: so, if a container with digest *DIGEST Y* is started under the image condition, it will then be stopped and deleted.
```

Comment:
```suggestion
As it can be seen, the *ContainerInstance2* digest is not included in the final allowlist, because it is disabled: so, if a container with digest *DIGEST Y* is started under the image condition, it will then be stopped and deleted.
```

What is the "image condition"?